src/test/regress/spec/isolation_get_distributed_wait_queries_mx.spec

```diff
@@ -275,7 +275,8 @@ permutation "s1-begin" "s1-update-ref-table-from-coordinator" "s2-start-session-
 
 // show that we can see blocking activity even if these are the first commands in the sessions
 // such that global_pids have not been assigned
-// in the second permutation, s3-show-actual-gpids shows the gpid for ALTER TABLE
-// because ALTER TABLE is not blocked on the parser but during the execution (hence gpid already asssigned)
-"s5-begin" "s5-alter" "s6-select" "s3-select-distributed-waiting-queries" "s3-show-actual-gpids" "s5-rollback"
-"s8-begin" "s8-select" "s7-alter" "s3-select-distributed-waiting-queries" "s3-show-actual-gpids" "s8-rollback"
+// in the first permutation, we would normally have s3-show-actual-gpids and it'd show the gpid has NOT been assigned
+// however, as of PG commit 3f323956128ff8589ce4d3a14e8b950837831803, isolation tester sends set application_name command
+// even before we can do anything on the session. That's why we removed s3-show-actual-gpids, but still useful to show waiting queriies
+permutation "s5-begin" "s5-alter" "s6-select" "s3-select-distributed-waiting-queries" "s5-rollback"
```

Comment:
and @halilozanakgul we seem to forget `permutation` which is weird how this worked properly :)

Reply:
If you look at the output you can find the answer, these two lines of steps were added to the end of the previous permutation

Reply:
that is true. it is an interesting way to avoid setup/teardown actually. I'll keep this mind for long running tests

Reply:
There are other ways to avoid setup/teardown. For example you can have setup/teardown steps for sessions. If you use per-session setup/teardown instead of per-test ones, you can easily avoid those.

This may not be always a nice idea, but it may work.

See PG docs for more: https://github.com/postgres/postgres/blob/7d12693473eca4008c5c53bb36a5f083068a941b/src/test/isolation/README#L83-L90